### PR TITLE
#8 Remove state check for upgrading

### LINKED ISSUE #8: Write persistent state to local config or remove if unnecessary

### PR DESCRIPTION
**Pull request by jelemux**
since that state is never set

Resolves #8

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- [#8] Upgrade base image to 3.20.1-1
```
Comment (kahoona77) [inline CHANGELOG.md:9]:
```suggestion
- [#8] Upgrade base image to 3.20.1-2
```

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM registry.cloudogu.com/official/base:3.15.8-1
+FROM registry.cloudogu.com/official/base:3.20.1-1
```
Comment (kahoona77) [inline Dockerfile:1]:
```suggestion
FROM registry.cloudogu.com/official/base:3.20.1-2
```

**Review by kahoona77 — CHANGES_REQUESTED** [review]
Update to use new doguctl v0.12.0

File: Dockerfile [hunk 3]
```diff
@@ -9,7 +9,7 @@ ENV SERVICE_TAGS=webapp \
     CONF_DIR=/usr/local/etc/redis \
     USER=redis \
     USER_ID=1000 \
-    REDIS_VERSION="6.2.12-r0" \
+    REDIS_VERSION="7.2.5-r0" \
```
Comment (ppxl) [inline Dockerfile:12]: Is this major version upgrade planned? It's a heavy thing to roll this up during a very simple change.

I know Redis 6.2 sucks but for this change I'd rather do the light lifting and the major upgrade in another change when we know the consequences with consuming dogus

Comment (ppxl) [reply Dockerfile]: Reduced to the latest available Redis version in the given alpine minor version

**Review by kahoona77 — APPROVED** [review]

**Review by kahoona77 — APPROVED** [review]
LGTM

**Pull request merged**